src/gitManager/gitManager.ts

```diff
@@ -300,4 +300,8 @@ export abstract class GitManager {
         }
         return template;
     }
+
+
+    abstract isFileTrackedByLFS(filePath: string): Promise<boolean>;
```

Comment:
No need to add it to `GitManager`, because there is no reason for an implementation in isomorphic-git, because it doesn't support git-lfs. So the method can only be implemented in `SimpleGit`